interfaces/repo_test.go

```diff
@@ -1539,7 +1539,7 @@ func (s *AddRemoveSuite) SetUpTest(c *C) {
 	c.Assert(err, IsNil)
 }
 
-func (s *AddRemoveSuite) TestAddSnapComplexErrorHandling(c *C) {
+/*func (s *AddRemoveSuite) TestAddSnapComplexErrorHandling(c *C) {
```

Comment:
Can you please remove the affected test function instead of commenting it out?

Reply:
Thanks for spotting... an oversight.